packaging/snapd.mk

```diff
@@ -72,8 +72,9 @@ $(builddir)/snap: GO_TAGS += nomanagers
 $(builddir)/snap $(builddir)/snap-seccomp $(builddir)/snapd-apparmor:
 	go build -o $@ $(if $(GO_TAGS),-tags "$(GO_TAGS)") \
 		-buildmode=pie \
-		-ldflags="-w -B 0x$$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \n')" \
```

Comment:
Would this affect new builds for other distros making them slightly larger? if they are not explicitly setting `EXTRA_GO_LDFLAGS =  -w` for example when building for ubuntu where `-w` was passed by default before but now isn't.

Reply:
My bad it seems that `snapd.mk` is only used by opensuse, so maybe adding `EXTRA_GO_LDFLAGS=-w` there is enough.